docs/en/Running-Spark-on-Alluxio.md

```diff
@@ -181,6 +187,8 @@ specify the address of Zookeeper in the URI in the format of "`zk@zkHost1:2181;z
 > double.saveAsTextFile("alluxio://zk@zkHost1:2181;zkHost2:2181/Output")
 ```
 
+This feature of encoding Zookeeper into Alluxio URI is not available on or before Alluxio 1.8.
```

Comment:
NIT: "is not available in versions 1.8 and earlier"

Reply:
fixed